### PR TITLE
Add INDEX space

### PR DESCRIPTION
**Pull request by inje**
**Add space for Index Coop**  
  
Etherscan: https://etherscan.io/address/0x0954906da0Bf32d5479e25f46056d22f08464cab
CoinGecko: https://www.coingecko.com/en/coins/index-cooperative
Website: https://www.indexcoop.com/
Twitter: https://twitter.com/indexcoop

### CODE REVIEW

**Review by e-asia — CHANGES_REQUESTED** [review]
You have some extra fields in your `index.json` file and your `skin.scss` has to be moved to the `skins` directory, the space creating process has been updated, you have to follow the new documentation. https://docs.snapshot.page/guides/create-a-space

**Comment by inje** [comment]
@e-asia Thanks for the reply. Which fields are the extra fields that should be removed?

**Comment by e-asia** [comment]
> @e-asia Thanks for the reply. Which fields are the extra fields that should be removed?

The `key` and `address` fields should be removed and the `skin` field is missing. Take a look on the sample file https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

File: spaces/index/index.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "strategies": [
+    [
+      "erc20-balance-of",
+      {
+        "address": "0x0954906da0Bf32d5479e25f46056d22f08464cab",
+        "symbol": "INDEX",
+        "decimals": 18
+      }
+    ]
+  ]
```
Comment (e-asia) [inline spaces/index/index.json:25]: This is the default strategies and it's useless to add it now, you should remove it.

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/index/index.json (hunk 1 above)
Comment (inje) [reply spaces/index/index.json]: Okay 👌. Updated

**Review by e-asia — APPROVED** [review]

**Pull request merged**